parachains/runtimes/collectives/collectives-polkadot/src/constants.rs

```diff
@@ -15,12 +15,17 @@
 
 pub mod account {
 	use frame_support::PalletId;
-	use sp_runtime::AccountId32;
 
 	/// Relay Chain treasury pallet id, used to convert into AccountId
 	pub const RELAY_TREASURY_PALL_ID: PalletId = PalletId(*b"py/trsry");
```

Comment:
If the other items are suffixed `_PALLET_ID`, this should be too for consistency.